### ISSUE TITLE
Remove `Get image description from Brave` settings from brave://settings/accessibility

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#45533]
### Description

Remove `Get image description from Brave` settings from brave://settings/accessibility 

### Steps to reproduce

1. Na

### Actual result

![Image](https://github.com/user-attachments/assets/b850a4a1-0785-4836-8404-ae31adaa5d3c)

### Expected result

`Get image description from Brave` settings should be removed from brave://settings/accessibility 

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.78.84 Chromium: 135.0.7049.100 (Official Build) beta (64-bit)
-- | --
Revision | 98cfa0e93c1b61488f7d0804518d360c197df69e
OS | Windows 10 Version 22H2 (Build 19045.5737)

### Channel information

- [ ] release (stable)
- [x] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc: @rebron @emerick @brave/qa-team

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
Brave | 1.78.88 Chromium: 136.0.7103.34 (Official Build) beta (64-bit)
-- | --
Revision | c3708d957bbde3ce51a507be8cdd52b9cb880ba3
OS | Windows 10 Version 22H2 (Build 19045.5737)
```
- Confirmed that `Get image description from Brave` settings is removed from brave://settings/accessibility

![Image](https://github.com/user-attachments/assets/d94423db-7736-4221-ae20-1b223559f98d)
